test/parallel/test-child-process-exec-maxBuffer.js

```diff
@@ -28,6 +28,25 @@ function checkFactory(streamName) {
   cp.exec(cmd, { maxBuffer: 5 }, checkFactory('stdout'));
 }
 
+// default value
+{
+  const cmd = `"${process.execPath}" -e "console.log('a'.repeat(200 * 1024))"`;
+
+  cp.exec(cmd, checkFactory('stdout'));
+}
+
+// default value
+{
+  // eslint-disable-next-line max-len
```

Comment:
This can be avoided by simply moving the string literal to the next line, indented at least two spaces.

Reply:
I thought one line is easier to read like the previous.
I changed it.